2025-10-17 22:56:49,030 INFO    MainThread:76 [wandb_setup.py:_flush():80] Current SDK version is 0.21.0
2025-10-17 22:56:49,030 INFO    MainThread:76 [wandb_setup.py:_flush():80] Configure stats pid to 76
2025-10-17 22:56:49,030 INFO    MainThread:76 [wandb_setup.py:_flush():80] Loading settings from /root/.config/wandb/settings
2025-10-17 22:56:49,030 INFO    MainThread:76 [wandb_setup.py:_flush():80] Loading settings from /take/hiratsuka/ResShift/wandb/settings
2025-10-17 22:56:49,030 INFO    MainThread:76 [wandb_setup.py:_flush():80] Loading settings from environment variables
2025-10-17 22:56:49,030 INFO    MainThread:76 [wandb_init.py:setup_run_log_directory():703] Logging user logs to /take/hiratsuka/ResShift/wandb/offline-run-20251017_225648-12hptehn/logs/debug.log
2025-10-17 22:56:49,030 INFO    MainThread:76 [wandb_init.py:setup_run_log_directory():704] Logging internal logs to /take/hiratsuka/ResShift/wandb/offline-run-20251017_225648-12hptehn/logs/debug-internal.log
2025-10-17 22:56:49,031 INFO    MainThread:76 [wandb_init.py:init():830] calling init triggers
2025-10-17 22:56:49,031 INFO    MainThread:76 [wandb_init.py:init():835] wandb.init called with sweep_config: {}
config: {'trainer': {'target': 'trainer.TrainerDifIR'}, 'model': {'target': 'models.unet.UNetModelSwin', 'ckpt_path': None, 'params': {'image_size': 64, 'in_channels': 3, 'model_channels': 160, 'out_channels': 3, 'attention_resolutions': [64, 32, 16, 8], 'dropout': 0, 'channel_mult': [1, 2, 2, 4], 'num_res_blocks': [2, 2, 2, 2], 'conv_resample': True, 'dims': 2, 'use_fp16': False, 'num_head_channels': 32, 'use_scale_shift_norm': True, 'resblock_updown': False, 'swin_depth': 2, 'swin_embed_dim': 192, 'window_size': 8, 'mlp_ratio': 4, 'cond_lq': True, 'lq_size': 64}}, 'diffusion': {'target': 'models.script_util.create_gaussian_diffusion', 'params': {'sf': 4, 'schedule_name': 'exponential', 'schedule_kwargs': {'power': 0.3}, 'etas_end': 0.99, 'steps': 15, 'min_noise_level': 0.04, 'kappa': 2.0, 'weighted_mse': False, 'predict_type': 'xstart', 'timestep_respacing': None, 'scale_factor': 1.0, 'normalize_input': True, 'latent_flag': True}}, 'autoencoder': {'target': 'ldm.models.autoencoder.VQModelTorch', 'ckpt_path': 'weights/autoencoder_vq_f4.pth', 'use_fp16': True, 'tune_decoder': False, 'params': {'lora_tune_decoder': False, 'embed_dim': 3, 'n_embed': 8192, 'ddconfig': {'double_z': False, 'z_channels': 3, 'resolution': 256, 'in_channels': 3, 'out_ch': 3, 'ch': 128, 'ch_mult': [1, 2, 4], 'num_res_blocks': 2, 'attn_resolutions': [], 'dropout': 0.0, 'padding_mode': 'zeros'}}}, 'degradation': {'sf': 4, 'resize_prob': [0.2, 0.7, 0.1], 'resize_range': [0.15, 1.5], 'gaussian_noise_prob': 0.5, 'noise_range': [1, 30], 'poisson_scale_range': [0.05, 3.0], 'gray_noise_prob': 0.4, 'jpeg_range': [30, 95], 'second_order_prob': 0.5, 'second_blur_prob': 0.8, 'resize_prob2': [0.3, 0.4, 0.3], 'resize_range2': [0.3, 1.2], 'gaussian_noise_prob2': 0.5, 'noise_range2': [1, 25], 'poisson_scale_range2': [0.05, 2.5], 'gray_noise_prob2': 0.4, 'jpeg_range2': [30, 95], 'gt_size': 256, 'resize_back': False, 'use_sharp': False}, 'data': {'train': {'type': 'PairedImageDataset', 'params': {'opt': {'dataroot_gt': '../ChestCT/HR', 'dataroot_lq': '../ChestCT/LR', 'dataroot_mask': '../ChestCT/mask', 'meta_info_file': 'data/meta_info/meta_info_ChestCT_val_2715_part_60.txt', 'io_backend': {'type': 'disk'}, 'gt_size': 256, 'use_hflip': True, 'use_rot': True, 'scale': 4, 'phase': 'train'}}}, 'val': {'type': 'PairedImageDataset', 'params': {'opt': {'dataroot_gt': '../ChestCT/HR', 'dataroot_lq': '../ChestCT/LR', 'dataroot_mask': '../ChestCT/mask', 'meta_info_file': 'data/meta_info/meta_info_ChestCT_val_2715_part_60.txt', 'io_backend': {'type': 'disk'}, 'scale': 4, 'phase': 'val'}}}}, 'train': {'lr': 5e-05, 'lr_min': 2e-05, 'lr_schedule': None, 'warmup_iterations': 5000, 'batch': [8, 1], 'microbatch': 8, 'num_workers': 4, 'prefetch_factor': 2, 'weight_decay': 0, 'ema_rate': 0.999, 'iterations': 300000, 'save_freq': 10000, 'log_freq': [200, 2000, 1], 'local_logging': True, 'tf_logging': False, 'use_ema_val': True, 'val_freq': 200, 'val_y_channel': True, 'val_resolution': 64, 'val_padding_mode': 'reflect', 'use_amp': True, 'seed': 123456, 'global_seeding': False, 'compile': {'flag': False, 'mode': 'reduce-overhead'}, 'scale': 4}, 'division': {'division_size': 64, 'overlap_size': 4}, 'logger': {'wandb': {'project': 'experiment5', 'name': 'ronbunnomama', 'id': None}}, 'save_dir': 'results', 'resume': '', 'cfg_path': 'configs/realsr_swinunet_realesrgan256.yaml', '_wandb': {}}
2025-10-17 22:56:49,031 INFO    MainThread:76 [wandb_init.py:init():871] starting backend
2025-10-17 22:56:49,258 INFO    MainThread:76 [wandb_init.py:init():874] sending inform_init request
2025-10-17 22:56:49,292 INFO    MainThread:76 [wandb_init.py:init():882] backend started and connected
2025-10-17 22:56:49,299 INFO    MainThread:76 [wandb_init.py:init():953] updated telemetry
2025-10-17 22:56:49,367 INFO    MainThread:76 [wandb_init.py:init():977] communicating run to backend with 90.0 second timeout
2025-10-17 22:56:56,112 WARNING MainThread:76 [wandb_init.py:init():1610] [no run ID] interrupted
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/wandb/sdk/lib/asyncio_compat.py", line 30, in run
    return future.result()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 439, in result
    self._condition.wait(timeout)
  File "/usr/lib/python3.8/threading.py", line 302, in wait
    waiter.acquire()
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/dist-packages/wandb/sdk/wandb_init.py", line 1606, in init
    return wi.init(run_settings, run_config, run_printer)
  File "/usr/local/lib/python3.8/dist-packages/wandb/sdk/wandb_init.py", line 996, in init
    result = wait_with_progress(
  File "/usr/local/lib/python3.8/dist-packages/wandb/sdk/mailbox/wait_with_progress.py", line 24, in wait_with_progress
    return wait_all_with_progress(
  File "/usr/local/lib/python3.8/dist-packages/wandb/sdk/mailbox/wait_with_progress.py", line 87, in wait_all_with_progress
    return asyncio_compat.run(progress_loop_with_timeout)
  File "/usr/local/lib/python3.8/dist-packages/wandb/sdk/lib/asyncio_compat.py", line 33, in run
    runner.cancel()
  File "/usr/lib/python3.8/concurrent/futures/_base.py", line 644, in __exit__
    self.shutdown(wait=True)
  File "/usr/lib/python3.8/concurrent/futures/thread.py", line 236, in shutdown
    t.join()
  File "/usr/lib/python3.8/threading.py", line 1011, in join
    self._wait_for_tstate_lock()
  File "/usr/lib/python3.8/threading.py", line 1027, in _wait_for_tstate_lock
    elif lock.acquire(block, timeout):
KeyboardInterrupt
2025-10-17 22:56:56,265 INFO    MsgRouterThr:76 [mailbox.py:close():129] [no run ID] Closing mailbox, abandoning 2 handles.
